### PR TITLE
Google Anaytics Source & Elixir Base

### PR DESCRIPTION
**Pull request by gordalina**
## What
Adds a native google-analytics source & Elixir base

## How
Native google-analytics source

## Pre-merge Checklist
* [ ] _Run integration tests_
* [ ] _Publish Docker images_

## Recommended reading order
1. `test.java`
1. `component.ts`
1. the rest

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1200367912513076/1200368078180428) by [Unito](https://www.unito.io)

### CODE REVIEW

**Comment by gordalina** [comment]
@sherifnada @michel-tricot opened a PR with the current state. I'll inform when the PR is close to being ready for review.

**Comment by sherifnada** [comment]
Thanks @gordalina ! I'll look into integrating with gradle

**Comment by gordalina** [comment]
@sherifnada feel free to add commits to this branch

**Comment by michel-tricot** [comment]
Thanks @gordalina ! Peeking already :)

**Comment by sherifnada** [comment]
@gordalina see https://github.com/gordalina/airbyte/pull/1 for an example addition of standard tests

File: airbyte-integrations/connectors/source-google-analytics/build.gradle [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+airbyteStandardSourceTestFile {
+    configPath = "secrets/config.json"
+    configuredCatalogPath = "priv/standard_source_tests/catalog.json"
+    specPath = "priv/connection_specification.json"
+}
+
+dependencies {
+    implementation files(project(':airbyte-integrations:bases:base-standard-fs-source-test').airbyteDocker.outputs)
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-google-analytics/build.gradle:17]: missed fixing this in my PR

```suggestion
    implementation files(project(':airbyte-integrations:bases:base-standard-source-test-file').airbyteDocker.outputs)
```

**Comment by sherifnada** [comment]
@gordalina still interested in contributing this one or can I close?

**Pull request closed without merging**